bundle/edu.gemini.pot/src/main/scala/edu/gemini/spModel/gemini/flamingos2/F2OiwfsProbeArm.scala

```diff
@@ -0,0 +1,111 @@
+      val (x1, y1) = (x0 + ProbeArmTaperedLength, -hm)
+      val (x2, y2) = (x0 + scaledLength, y1)
+      val (x3, y3) = (x2, hm)
+      val (x4, y4) = (x1, y3)
+      val (x5, y5) = (x0, htw)
+
+      val points = List((x0, y0), (x1, y1), (x2, y2), (x3, y3), (x4, y4), (x5, y5))
+      ImPolygon(points)
+    }
+
+    val pickoffMirror: Shape = {
+      val plateScale = flamingos2.getLyotWheel.getPlateScale
+      val scaledMirrorSize = PickoffMirrorSize * plateScale
+      val xy = -scaledMirrorSize / 2.0
+      new Rectangle2D.Double(xy, xy, scaledMirrorSize, scaledMirrorSize)
+    }
+
+    List(probeArm, pickoffMirror)
+  }
+
+  override def armAdjustment(ctx0: ObsContext,
+                             guideStarCoords: Coordinates,
+                             offset0: Offset,
+                             T: Point2D): Option[ArmAdjustment] = {
+    import ProbeArmGeometry._
+
+    for {
+      ctx      <- Option(ctx0)
+      offset   <- Option(offset0)
+      gemsFlag = ctx.getAOComponent.asScalaOpt.fold(false){ ao =>
+        val aoNarrowType = ao.getNarrowType
+        aoNarrowType.equals(Gems.SP_TYPE.narrowType)
+      }
+    } yield {
+      val flamingos2  = ctx.getInstrument.asInstanceOf[Flamingos2]
+      val flip        = if (flamingos2.getFlipConfig(gemsFlag)) -1 else 1
+      val posAngle    = ctx.getPositionAngle.toRadians.getMagnitude
+      val fovRotation = flamingos2.getRotationConfig(gemsFlag).toRadians.getMagnitude
+      val guideStarPt = guideStarPoint(ctx, guideStarCoords)
+      val angle       = armAngle(posAngle, fovRotation, guideStarPt, T, flip, flamingos2.getLyotWheel.getPlateScale)
+      ArmAdjustment(angle, guideStarPt)
+    }
+  }
+
+  private def armAngle(posAngle:    Double,
+                       fovRotation: Double,
+                       guideStar:   Point2D,
+                       T:           Point2D,
+                       flip:        Int,
+                       plateScale:  Double): Angle = {
+    val P = {
+      val scaledPAO = ProbeArmOffset * plateScale
+      val angle     = -posAngle - fovRotation
+      new Point2D.Double(scaledPAO * flip * math.cos(angle), scaledPAO * flip * math.sin(angle))
+    }
+    val D = new Point2D.Double(guideStar.getX + T.getX - P.getX, guideStar.getY + T.getY - P.getY)
+
+    val Q = {
+      val scaledPBAL = ProbeBaseArmLength * plateScale
+      val scaledPPAL = ProbePickoffArmLength * plateScale
+      val distance   = math.min(math.sqrt(D.getX * D.getX + D.getY * D.getY), scaledPBAL + scaledPPAL)
+
+      val a = (scaledPBAL * scaledPBAL - scaledPPAL * scaledPPAL + distance * distance) / (2 * distance)
+      val h = flip * math.sqrt(scaledPBAL * scaledPBAL - a * a)
+
+      new Point2D.Double(-guideStar.getX - T.getX + P.getX + (a * D.getX + h * D.getY) / distance,
+                         -guideStar.getY - T.getY + P.getY + (a * D.getY - h * D.getX) / distance)
+    }
+
+    val angle = Angle.fromRadians(math.atan2(Q.getY, Q.getX))
```

Comment:
Good call. I was using it formerly in a println for comparison and forgot that I didn't require the temporary variable any more.